vcf/parser.py

```diff
@@ -654,30 +651,30 @@ def __init__(self, stream, template, lineterminator="\n"):
         # get a maximum key).
         self.info_order = collections.defaultdict(
             lambda: len(template.infos),
-            dict(zip(template.infos.iterkeys(), itertools.count())))
+            dict(list(zip(iter(list(template.infos.keys())), itertools.count()))))
```

Comment:
Ugh. I'm sure this can be rewritten better, but if it's what the tool says, I guess it's fine for now.